lighthouse-extension/gulpfile.js

```diff
@@ -14,7 +14,6 @@ const eslint = require('gulp-eslint');
 const livereload = require('gulp-livereload');
 const tap = require('gulp-tap');
 const zip = require('gulp-zip');
-const rename = require('gulp-rename');
 
 const audits = fs.readdirSync(path.join(__dirname, '../', 'lighthouse-core/audits/'))
```

Comment:
note to me/you/all that we'll need to update this to hand the dbw directories. similarly with gatherers.